jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1597,6 +1597,10 @@ properties:
             title: Source Photometry and Morphology
             type: string
             fits_keyword: S_SRCCAT
+          guider_cds:
+            title: Guider CDS Correction
```

Comment:
Minor nit, but I would call this "Guider CDS **Calculation**", as opposed to Correction, because it isn't really correcting for anything, just computing differences.